tests/ui/from_instead_of_into.stderr

```diff
@@ -0,0 +1,78 @@
+error: Into trait is preferable than From as a generic bound
+  --> $DIR/from_instead_of_into.rs:11:5
+   |
+LL |     u32: From<T>,
+   |     ^^^^^^^^^^^^
+   |
+   = note: `-D clippy::from-instead-of-into` implied by `-D warnings`
+help: remove From bound
+   |
+LL |     ,
+   |    --
```

Comment:
This is probably not the best diagnostic message, `From<T>` is removed and only comma left.

Reply:
Yeah, I don't even know if that still compiles. I think you have to track how many bounds there are and tweak the suggestion depending on this.